creator/studies/nodes.py

```diff
@@ -31,6 +32,12 @@ class StudyNode(DjangoObjectType):
         description="List all releases for the study",
     )
 
+    versions = DjangoFilterConnectionField(
```

Comment:
Why was this needed? Did it not allow you to sort by version properties without it?

Reply:
yeah exactly